packages/slice-machine/src/hooks/useMarketingContent.ts

```diff
@@ -53,6 +69,12 @@ const DEFAULT_CONTENT: MarketingContent = {
         "Follow our Quick Start guide to learn the basics of Slice Machine",
     },
   },
+  masterSliceLibrary: {
```

Comment:
And so nothing needed here, related to my other comment above

Reply:
Okay, then in the component I have this:
```
const { masterSliceLibrary } = useMarketingContent();
const { exampleLinkUrl, codeLinkUrl, previewVideoUrl } = masterSliceLibrary;
```
which will not compile with that change.

How would you prefer me to address that? I could use `masterSliceLibrary!` because if the modal is displayed we can be sure those values are set, or I could do this:
```
const { masterSliceLibrary } = useMarketingContent();

if (!masterSliceLibrary) return null;

const { exampleLinkUrl, codeLinkUrl, previewVideoUrl } = masterSliceLibrary;
```
Do you have any preference?

Reply:
Ah I see! 
Hum, for me a mix of both I would say then. Good catch!

So for me you could continue to define a type and then pass it as a props:

```ts
type MasterSliceLibraryPreviewModalProps = {
  isOpen: boolean;
  masterSliceLibrary: MasterSliceLibraryMarketingContent;
  onClose: () => void;
};
```
Then before:

```ts
<SideNavList position="bottom">
        {masterSliceLibrary !== undefined && (
          <SideNavListItem>
            <MasterSliceLibraryPreviewModal
              isOpen={isSliceLibraryDialogOpen}
              masterSliceLibrary={masterSliceLibrary}
              onClose={() => {
                setIsSliceLibraryDialogOpen(false);
              }}
            />
            <SideNavLink
              title="Master Slice Library"
              href={"/"}
              Icon={MasterSliceLibraryIcon}
              onClick={(e) => {
                void telemetry.track({
                  event: "slice-library:beta:modal-opened",
                });

                setIsSliceLibraryDialogOpen(true);

                // We don't want it to actually navigate anywhere, but `href` is required
                e.preventDefault();
              }}
            />
          </SideNavListItem>
        )}
```

Reply:
WDYT?

Reply:
Personally, I'd prefer to not pass this information as props from the outside and keep getting the marketing content inside the component, but if that's what you prefer I can change the code like this.

Reply:
Then I prefer, like this 🙂

```
const { masterSliceLibrary } = useMarketingContent();

if (!masterSliceLibrary) return undefined; // we prefer undefined, but sometimes React don't want 

const { exampleLinkUrl, codeLinkUrl, previewVideoUrl } = masterSliceLibrary;
```

Reply:
It's just a double check that masterSliceLibrary is here, with the parent call but it's good for me 🙌

Reply:
As discussed in DMs, went with early return option.